src/mixins/filing-template-mixin.ts

```diff
@@ -1461,8 +1461,7 @@ export default class FilingTemplateMixin extends DateMixin {
   // FUTURE: probably don't need this becayse Corrected Filing Date is already YYYY-MM-DD
   /** The corrected filing date as YYYY-MM-DD. */
   private get correctedFilingDate (): string {
-    const date = this.apiToDate(this.getCorrectedFilingDate)
-    return this.dateToYyyyMmDd(date)
+    return this.getCorrectedFilingDate
```

Comment:
Essentially, this getter does nothing. I recommend just using `getCorrectedFilingDate` in the 2 places this getter is used?

Also, could you please correct the comment on this property? Thanks.

![image](https://github.com/user-attachments/assets/45878276-8209-4250-a2af-46c1a3ebc715)